backend/prompt_studio/prompt_studio_registry_v2/fields.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from django.db import models
+
+logger = logging.getLogger(__name__)
+
+
+class ToolPropertyJSONField(models.JSONField):
+    def from_db_value(self, value, expression, connection):  # type: ignore
+        metadata = super().from_db_value(value, expression, connection)
+        return metadata
+
+
+class ToolSpecJSONField(models.JSONField):
+    def from_db_value(self, value, expression, connection):  # type: ignore
+        metadata = super().from_db_value(value, expression, connection)
+        return metadata
+
+
+class ToolVariablesJSONField(models.JSONField):
+    def from_db_value(self, value, expression, connection):  # type: ignore
+        metadata = super().from_db_value(value, expression, connection)
+        return metadata
+
+
+class ToolMetadataJSONField(models.JSONField):
+    def from_db_value(self, value, expression, connection):  # type: ignore
+        metadata = super().from_db_value(value, expression, connection)
+        return metadata
```

Comment:
@muhammad-ali-e can do some refactoring here to avoid code repetition?

Reply:
Updated with a base class along a TODO comment